plugin-api/src/main/java/org/sonar/api/utils/log/LoggersLoader.java

```diff
@@ -19,24 +19,22 @@
  */
 package org.sonar.api.utils.log;
 
-class ConsoleLoggers extends Loggers {
+import java.util.ServiceLoader;
 
-  private LoggerLevel level = LoggerLevel.INFO;
+class LoggersLoader {
 
-  @Override
-  protected Logger newInstance(String name) {
-    return new ConsoleLogger();
-  }
+  private static final Loggers INSTANCE = ServiceLoader.load(Loggers.class).findFirst().orElse(new Slf4jLoggers());
```

Comment:
what's the intended use case for providing an alternative implementation of `Loggers`?  (less is more)

Reply:
In order to support the `setLevel`, which is not supported by slf4j. If we decide to drop this use case, or find a better way, we would be able to drop that.